frontend/src/metabase/components/HeaderBar.jsx

```diff
@@ -1,53 +1,39 @@
+/* @flow */
 import React, { Component } from "react";
 
 import Input from "metabase/components/Input.jsx";
 import TitleAndDescription from "metabase/components/TitleAndDescription.jsx";
 
-import cx from "classnames";
-
-export default class Header extends Component {
+export default class HeaderBar extends Component {
```

Comment:
FYI, not sure if it was your intention but props/state don't get typechecked by flow unless you do `Component<*, Props, State>`